sdk/cosmos/azure-cosmos/azure/cosmos/aio/_database.py

```diff
@@ -547,10 +685,11 @@ async def replace_container(
         :keyword Dict[str, Any] full_text_policy: **provisional** The full text policy for the container.
             Used to denote the default language to be used for all full text indexes, or to individually
             assign a language to each full text index path.
-        :returns: A `ContainerProxy` instance representing the container after replace completed.
+        :returns: A `ContainerProxy` instance representing the new container or a tuple of the ContainerProxy
+            and CosmosDict with the response headers.
         :raises ~azure.cosmos.exceptions.CosmosHttpResponseError: Raised if the container couldn't be replaced.
             This includes if the container with given id does not exist.
-        :rtype: ~azure.cosmos.aio.ContainerProxy
+        :rtype: ~azure.cosmos.ContainerProxy or tuple[ContainerProxy, CosmosDict]
```

Comment:
These references will need to be fully qualified or the docs wont be linked:
```suggestion
        :rtype: ~azure.cosmos.ContainerProxy or tuple[~azure.cosmos.ContainerProxy, ~azure.cosmos.CosmosDict]
```